src/main/java/dev/snowdrop/release/services/GitService.java

```diff
@@ -227,7 +227,7 @@ public String getBranch() {
         }
 
         protected String defaultBranch() {
-            return "master";
+            return "main";
```

Comment:
Note that this change will need to be propagated to #114 as well

Reply:
The #114 branch needs to be rebased onto main